src/reduxStore.js

```diff
@@ -53,7 +53,7 @@ const getReduxStore = () => {
       }
 
       persistStore(store,
-        { whitelist: ['certificate'] },
+        { whitelist: ['certificate', 'banner'] },
```

Comment:
it will just sit there forever, thoughts on if i should handle this?

Reply:
I think you can supply `autoRehydrate()` with a customized config which contains a customized `stateReconciler` so you can preprocess the banner states before they made their way back to redux https://www.npmjs.com/package/redux-persist/v/4.6.0-es#autorehydrateconfig
But it might be complex to do, and there might be more elegant way exists in `redux-persist` V5+ ...
So I'm also okay if we take this `resetDate` feature out from this PR for now and backlog a new ticket for it

Reply:
after looking into i added code to TopBar to look through the store and remove expired banners